### PR TITLE
Fix random lint issues

### PR DESCRIPTION
**Pull request by knorrium**
These should silence some of the warnings we see in PRs

### CODE REVIEW

File: backend/src/api/audit.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ class Audit {
           if (fits) {
             lastFeeRate = Math.min(lastFeeRate, tx.effectiveFeePerVsize);
           }
-          if (tx.firstSeen == null || (now - (tx?.firstSeen || 0)) > PROPAGATION_MARGIN) {
+          if (tx.firstSeen === null || (now - (tx?.firstSeen || 0)) > PROPAGATION_MARGIN) {
```
Comment (softsimon) [inline backend/src/api/audit.ts:74]: Afraid this and that other case might be `undefined` here, changing to `!==` will break it .. maybe  @mononaut  can confirm.

Comment (mononaut) [reply backend/src/api/audit.ts:74]: yeah, non-strict null equality checks are usually deliberate.

can we change the `eqeqeq` linting rule to `["error", "always", {"null": "ignore"}]`?

**Review by softsimon — CHANGES_REQUESTED** [review]
Comments

**Comment by softsimon** [comment]
Closing due to low activity

**Pull request closed without merging**